manifests/integrations/php_fpm.pp

```diff
@@ -21,12 +21,25 @@
 #
 
 class datadog_agent::integrations::php_fpm(
+  $http_host        = undef,
```

Comment:
Small nit (since all parameters are essentially optional, it doesn't make much of a difference), we probably want to `http_host` to be placed _after_ `ping_url`, just to resemble the YAML structure and the fact that it really can be `undef` whereas the other two are a must.